power_grid_model_io_native_c/power_grid_model_io_native_c/src/vnf_pgm_converter.cpp

```diff
@@ -4,29 +4,49 @@
 
 #define PGM_IO_DLL_EXPORTS
 
+#include <power_grid_model_io_native/common/enum.hpp>
 #include <power_grid_model_io_native/vnf_converter/vnf_pgm_converter.hpp>
 
 #include "handle.hpp"
 #include <power_grid_model_io_native_c/basics.h>
 #include <power_grid_model_io_native_c/vnf_pgm_converter.h>
 
 #include <power_grid_model/auxiliary/dataset.hpp>
+#include <power_grid_model/common/exception.hpp>
 
-using power_grid_model::ConstDataset;
+using namespace power_grid_model_io_native;
```

Comment:
whoops, i missed this one. instead of `using namespace power_grid_model_io_native; /*code*/`, please use a namespace alias like `namespace pgm_io = power_grid_model_io_native;` and then in the code use `pgm_io::` everywhere. `using namespace` has the awkward side-effect of placing things in a global namespace (unless you in turn do it in an anonymous namespace `namespace { /*code*/ }`, but that's going into details). Especially when dealing with an `extern` functionality, namespaces are your friend.

Reply:
will do